### PR TITLE
The boundary of the Moebius band, and fixes for V types.

### PR DESCRIPTION
**Pull request by favonia**

### CODE REVIEW

File: example/moebius-boundary.red [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+let moebius-boundary→s1 (x : moebius-boundary) : s1 =
+  moebius-boundary→s1' (x .fst) (x .snd)
+
+;let s1→moebius-boundary (x : s1) : moebius-boundary =
+;  elim x [
+;  | base → (base, tt)
+;  | loop i →
+;      (comp 0 1 (loop i) [i=0 → refl | i=1 → λ j → loop j],
+;       box 0 1 (vin i tt ff) [i=0 → tt | i=1 → tt])
```
Comment (favonia) [inline example/moebius-boundary.red]: (This is based on some hypothetical syntax.)

File: example/moebius-boundary.red [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+let s1→moebius-boundary→s1
+  : path s1 (moebius-boundary→s1 (s1→moebius-boundary base)) base =
+  refl
+
+-- there is an invalid fhcom in the middle?!
+-- ... (fhcom 0 1 (loop x) [x=0 <x1> base]) ...
+let test : dim → s1 =
+  λ i → moebius-boundary→s1 (s1→moebius-boundary (loop i))
+normalize test
```
Comment (favonia) [inline example/moebius-boundary.red]: Serious, what was going on here?

File: example/moebius-boundary.red [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+/-
+let double : s1 → s1 = λ x → s1→moebius-boundary x .fst
+
+import omega1s1
+
+let test0 : path int (winding (λ i → double (loopn (pos (suc zero)) i))) (pos (suc (suc zero))) = refl
```
Comment (jonsterling) [inline example/moebius-boundary.red]: btw I was just playing around and this commented out code works

Comment (favonia) [reply example/moebius-boundary.red]: Yes, it should, but it will be removed eventually.

File: example/moebius-boundary.red [hunk 4]
```diff
@@ -80,6 +80,12 @@ opaque let s1→moebius-boundary→s1/loop :
 
 quit
 
+/-
+  This will force re-typechecking `box`, but why?
```
Comment (jonsterling) [inline example/moebius-boundary.red]: can you say what you mean by this?

**Pull request merged**